pkg/ocm/sca/sca.go

```diff
@@ -221,30 +223,34 @@ func (c *Controller) requestSCAWithExpBackoff(ctx context.Context, endpoint stri
 	}
 
 	klog.Infof("Nodes architectures: %s", architectures)
-	var data []byte
+	var responses [][]byte
+	responses = make([][]byte, len(architectures))
 	err := wait.ExponentialBackoff(bo, func() (bool, error) {
-		var err error
-		data, err = c.client.RecvSCACerts(ctx, endpoint)
-		if err != nil {
-			// don't try again in case it's not an HTTP error - it could mean we're in disconnected env
-			if !insightsclient.IsHttpError(err) {
-				return true, err
-			}
-			httpErr := err.(insightsclient.HttpError)
-			// try again only in case of 500 or higher
-			if httpErr.StatusCode >= http.StatusInternalServerError {
-				// check the number of steps to prevent "timeout waiting for condition" error - we want to propagate the HTTP error
-				if bo.Steps > 1 {
-					klog.Errorf("%v. Trying again in %s", httpErr, bo.Step())
-					return false, nil
+		for arch := range architectures {
```

Comment:
I think we don't need to do this. I think we can append the architecture to one string - e.g "x86_64,aarch64"

Reply:
Then, we will need to split the string and iterate in the `insightsclient.Client.RecvSCACerts` method, as the used API only allow to perform 1 request per architecture

Reply:
I have checked the `certificates` API and it doesn't support this kind of request where the architecture is a string with multiple values separated by commas.

I tried with an array of strings too, and it doesn't work neither (different API errors), so I guess the only way to implement this is retrieving one certificate at a time